modules/fedimint-mint-client/src/lib.rs

```diff
@@ -314,10 +314,11 @@ impl FromStr for OOBNotes {
 
     /// Decode a set of out-of-band e-cash notes from a base64 string.
     fn from_str(s: &str) -> Result<Self, Self::Err> {
-        let bytes = if let Ok(bytes) = BASE64_URL_SAFE.decode(s) {
+        let s: String = s.chars().filter(|&c| !c.is_whitespace()).collect();
```

Comment:
Why not `let s = s.trim()`? Are whitespaces in the middle really a problem?

Reply:
Yeap. Define "really". :D

I often lamely copy&paste between terminals using mouse selection and it just adds `\n` on each line. To me it's real 1st world problem. But I can imagine people copy&pasting these between emails and other mediums that can introduce newlines, and don't see a good reason why not just do the right thing.

BTW. There's a precedent. As `base64` originally used to encode binary data in emails it adds newlines:

```
> dd if=/dev/urandom count=200 bs=1 2>/dev/null | base64
Bw6feHFDjJ5JHeFB9eUPjQqDZvCq712eOS2CUTHBtjbSUUq8nPolBCVXckW0/TzfEhXNwd+8treJ
aB91yNYHNRndPPT1M20hKxY+ZMSxJHCTbCiPLZ/H4cApf+HSjOvZXCgPU4LRJOyRJBrWL8ZykAPx
Dap0Uzao9tITxLCX4Nweyy9qMoX5s/0u0wic4mQUID2IiiGyDyALQOTkAlRnwQ+kcFeylg+X77P1
149K+C8BFAw8xy/yBTLdlKYJOU0105ZZ7MkHtYE=
```

And ignores whitespace just automatically:

```
> dd if=/dev/urandom count=200 bs=1 2>/dev/null | base64 | base64 -d | base64
kUg1xjRwrOT9cFiIrJeIyTIo/S1zYl5jXmhb6dbmYjtHOpBIMRXGYEKjspH3SZ6GznuKr4k/Q1lG
WCeTG4BtQX9ky3m1E4QIPpla780tbg4w+HSamY9mogLK13UK4P4NJWzhe07vFza0bJ5J3HcOlK9H
5OqjF9J8OaxxfFUzaYOsSqOnSizOmDDrT7X8Ull/9OZzBt6IZFGlid0/rAimWmeqGoy/Vb57JN1F
En/BZZPxY6bYlh9akwt4f/dJosiqgjK1sylyu4s=
```

@elsirion